renpy/config.py

```diff
@@ -1177,6 +1177,9 @@ def say_attribute_transition_callback(*args):
 # Should Ren'Py raise exceptions when finding an image?
 raise_image_exceptions = True
 
+# Should the size transform property only accept numbers of pixels ?
+old_size_tpref = False
```

Comment:
This should probably be positive (defaulting to true), and tpref is something that's only used in the documentation - it's short for transform property reference. How about:

```
relative_transform_size = True
```